libraries/aws_backend.rb

```diff
@@ -148,6 +148,18 @@ def initialize(opts)
       client_args[:client_args][:region] = opts[:aws_region] if opts[:aws_region]
       # below allows each resource to optionally and conveniently set an endpoint
       client_args[:client_args][:endpoint] = opts[:aws_endpoint] if opts[:aws_endpoint]
+      # below allows each resource to optionally and conveniently set max_retries
+      if opts[:retry_limit]
+        client_args[:client_args][:retry_limit] = opts[:retry_limit].to_i
+      elsif ENV['retry_limit']
+        client_args[:client_args][:retry_limit] = ENV['retry_limit'].to_i
```

Comment:
Can we make this case insensitive `AWS_RETRY_LIMIT` (and `AWS_RETRY_BACKOFF` below) to be consistent with other variables?